tests/testthat/test-minimal_example.R

```diff
@@ -0,0 +1,48 @@
+require(dplyr)
+require(jsonlite)
+require(readr)
+
+dir <- "test_data/minimal_"
+# dir <- "tests/testthat/test_data/minimal_"
+
+data <- readr::read_tsv(paste0(dir, "data.tsv")) %>%
+  mutate_at(
+    c("categories1", "categories2", "categories3"),
+    function(column) {
+      lapply(column, function(val) {
+        unlist(jsonlite::fromJSON(val))
+      })
+    }
+  ) %>%
+  mutate_at(vars(ends_with("_str")), as.character)
+column_info <- readr::read_tsv(paste0(dir, "column_info.tsv"))
+column_groups <- readr::read_tsv(paste0(dir, "column_groups.tsv"))
+row_info <- readr::read_tsv(paste0(dir, "row_info.tsv"))
+row_groups <- readr::read_tsv(paste0(dir, "row_groups.tsv"))
+palettes <- jsonlite::read_json(paste0(dir, "palettes.json"), simplifyVector = TRUE)
+palettes$pie <- unlist(palettes$pie)
+
+legends <- list(
+  list(
+    title = "Text",
+    palette = "black_text",
+    enabled = TRUE,
+    geom = "text",
+    labels = c("propA", "propB", "propC"),
+    values = c("Property of A", "Property of B", "Property of C")
+  )
+)
+
+test_that("minimal example funky_heatmap works", {
+  g <- funky_heatmap(
```

Comment:
I changed this from `funkyheatmap::funky_heatmap` to this, because otherwise the unit test will run the installed version of funkyheatmap and not the current source code, since testthat uses something similar to `devtools::load_all()` to load the current source code.

Reply:
Oh, that's useful to know, thanks!